examples/kitchen-sink/apps/admin/src/main.tsx

```diff
@@ -1,11 +1,16 @@
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import { createRoot } from "react-dom/client";
+import App from "./app";
 import "./index.css";
-import App from "./App";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
-);
+const el = document.getElementById("root");
+if (el) {
+  const root = createRoot(el);
```

Comment:
Just for my own curiosity, is this the suggested way to do things these days?

Reply:
@chris-olszewski indeed, this changed with react-dom 18 ([docs](https://react.dev/reference/react-dom/client/createRoot#createroot))